test/runtests.py

```diff
@@ -99,9 +103,19 @@ def test_path(folder, is_file):
                 baseline = os.path.splitext(js_file)[0] + '.baseline'
                 baseline = os.path.join(folder, baseline)
                 if os.path.isfile(baseline):
-                    expected_output = None
+                    baseline_output = None
                     with open(baseline, 'r') as bs_file:
-                        expected_output = bs_file.read().replace('\r', '')
+                        baseline_output = bs_file.read()
+
+                    # clear carriage return
+                    # todo: remove carriage return at the end of the line
+                    #       or better fix ch to output same on all platforms
+                    expected_output = baseline_output.replace('\r', '')
+
+                    # normalize tabs on both outputs to 2 spaces
+                    expected_output = expected_output.replace('\t', '  ')
```

Comment:
> The difference is due to output from ch on a particular platform. This issue is similar to \r

What's the difference of `\t` on different platforms? Isn't it the same character?

You replaced here only in the log, but actual `.baseline` file contains `\t`. Thus people will be confused reading the log file and the baseline file as they don't match.